### PR TITLE
responsive for registration input

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**
# Description

Responsiveness for Registration input 

Fixes # (#100)

# How to test?

npm run storybook

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/RegistrationPage/RegistrationPage.styles.css [hunk 1]
```diff
@@ -81,3 +81,7 @@
   width: 50%;
   padding: 2px 10px;
 }
+
+.email_container input {
+  width: 25vw;
```
Comment (dpfernandes) [inline src/client/containers/RegistrationPage/RegistrationPage.styles.css:86]: Please work with percentage (%) like you did with the others css classes

**Review by dpfernandes — CHANGES_REQUESTED** [review]

**Comment by soujanyapolavarapu** [comment]
@dpfernandes Please check and approve this PR i believe, it can be merge now.  :)

**Review by dpfernandes — APPROVED** [review]
👍

**Review by rajanibusani — APPROVED** [review]
It is Responsive in all screens now...

**Pull request merged**